assets/collage.css

```diff
@@ -111,14 +111,18 @@
 }
 
 .collage-card {
-  background: rgb(var(--color-background));
   height: 100%;
   position: relative;
   border-radius: var(--border-radius);
   border: var(--border-width) solid rgba(var(--color-foreground), var(--border-opacity));
   padding: var(--image-padding);
 }
 
+.collage-card.gradient {
+  /* Needed for gradient continuity with or without animation */
```

Comment:
For `2.` the reason I didn't do that (even though I think ideally it's better), is because sometimes it's a whole declaration that's the fix but sometimes it's just one declaration and not the whole thing. For example: 
```css
.collapsible-row-layout .accordion {
  border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
  margin-bottom: 1.5rem;
  /* Needed for gradient continuity with or without animation */
  background-attachment: local;
```

So I went with a similar approach everywhere but could mix it up where we can if you think it's better 👍